tests/Query/ExpressionTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ClickHouseDB\Tests\Query;
+
+use ClickHouseDB\Query\Expression;
+use ClickHouseDB\Quote\FormatLine;
+use PHPUnit\Framework\TestCase;
+
+class ExpressionTest extends TestCase
+{
+    public function testToString()
+    {
+        $expressionString = "UUIDStringToNum('0f372656-6a5b-4727-a4c4-f6357775d926')";
+        $expr = new Expression($expressionString);
+
+        $this->assertEquals(
+            $expressionString,
+            (string) $expr
+        );
+    }
+
+    public function testExpressionValueForInsert()
+    {
+        $expressionString = "UUIDStringToNum('0f372656-6a5b-4727-a4c4-f6357775d926')";
+        $preparedValue = FormatLine::Insert([new Expression($expressionString)]);
+
+        $this->assertEquals(
```

Comment:
```suggestion
        self::assertEquals(
```

Reply:
Why `self` is better then `$this`?

Reply:
I tried to explain it here few days ago https://github.com/webonyx/graphql-php/pull/354#issuecomment-422900157 

TLDR: the method is a static wrapper and therefore should be called like every other static method you call statically, not dynamically.

Reply:
Haven't looked in phpunit sources that carefully. I think it came from phpunit docs. Or at least from early ones.
Thanks for link with deep explanation.